src/System.Net.NameResolution/tests/PalTests/project.json

```diff
@@ -24,12 +24,16 @@
     "System.Text.RegularExpressions": "4.2.0-beta-24403-05",
     "System.Threading": "4.0.12-beta-24403-05",
     "System.Threading.Tasks": "4.0.12-beta-24403-05",
+    "System.Net.Sockets": {
```

Comment:
This sounds similar to the issues we're having when testing System.dll with VSTest that references System.dll.
In that case what we do is to use `#pragma warning disable 0436` to avoid the warnings. In the case of System.Net.Internal types, the ambiguity comes from having both using statements:
```
using System.Net.Internals;
using System.Net.Sockets;
```

@stephentoub @ericeil